frontend/src/scenes/onboarding/OnboardingBillingStep.tsx

```diff
@@ -40,20 +43,21 @@ export const OnboardingBillingStep = ({
                         // TODO: redirect path won't work properly until navigation is properly set up
                         to={getUpgradeProductLink(product, plan.plan_key || '', redirectPath, true)}
                         type="primary"
+                        status="alt"
                         center
                         disableClientSideRouting
                         onClick={() => {
                             reportBillingUpgradeClicked(product.type)
                         }}
                     >
-                        Subscribe
+                        Subscribe to Paid Plan
```

Comment:
No need to do this now, but this feels like a good A/B test to run- subscribe vs upgrade vs add credit card

Reply:
Added to kanban